chore: fix lint errors (#8183)

Payal-Goswami opened the pull request:
Resolves #8183.

## Description

This pull request fixes:

- File: lib/node_modules/@stdlib/utils/nonenumerable-properties-in/lib/main.js

Added a missing require statement for Object to satisfy the stdlib/require-globals lint rule.

- File: lib/node_modules/@stdlib/stats/ztest/examples/index.js

Replaced the prohibited new Array() constructor with an array literal and loop push.

## Related Issues

This pull request:

-   resolves #8183

## Questions

No.

## Other

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

Payal-Goswami committed:
fix: require Object + remove new Array (lint #8183)

Payal-Goswami committed:
fix-lint-errors-index.js-#8183

kgryte committed:
fix: remove nested import

Signed-off-by: Athan <kgryte@gmail.com>

kgryte committed:
docs: update example

Signed-off-by: Athan <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/stats/ztest/examples/index.js:
@Payal-Goswami Would you mind making this same change to the corresponding README? The example is duplicated there.

Payal-Goswami replied:
@kgryte Got it 👍, I’ll update the README accordingly.

Payal-Goswami replied:
@kgryte Updated the README examples to match the latest index.js changes as suggested.

Payal-Goswami committed:
merge code from upstream
Merge branch 'develop' of https://github.com/stdlib-js/stdlib into fix-lint-errors-8183

Payal-Goswami committed:
docs: Update README examples to match index.js

Payal-Goswami committed:
Merge changes
Merge branch 'fix-lint-errors-8183' of https://github.com/Payal-Goswami/stdlib into fix-lint-errors-8183

Payal-Goswami committed:
Updated comment in index.js

kgryte committed:
docs: update examples

Signed-off-by: Athan <kgryte@gmail.com>

kgryte committed:
docs: update examples

Signed-off-by: Athan <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/stats/ztest/examples/index.js:
```diff
@@ -18,24 +18,14 @@
+var out = ztest( arr, 4.0 );
 console.log( out.print() );
 
 // Test whether true mean is equal to five:
```
```suggestion
// Test whether true mean is equal to 5.0:
```

kgryte committed:
docs: update comment

Signed-off-by: Athan <kgryte@gmail.com>

kgryte committed:
docs: update examples

Signed-off-by: Athan <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/stats/ztest/README.md:
```diff
@@ -35,24 +35,18 @@ var ztest = require( '@stdlib/stats/ztest' );
+// Create an array of random numbers:
+var arr = normal( 300, 0.0, 2.0 );
 
+// Test whether true mean is equal to 2.0:
```
```suggestion
// Test whether true mean is equal to 0.0:
```

kgryte committed:
docs: fix comment

Signed-off-by: Athan <kgryte@gmail.com>

Payal-Goswami commented:
Thanks for merging! 🙌 I appreciate the adjustments you made - I’ll take note of them when working on future contributions.
